rero_ils/modules/holdings/api.py

```diff
@@ -148,6 +148,20 @@ def get_holdings_pid_by_document_pid_by_org(cls, document_pid, org_pid):
         for holding in results:
             yield holding.pid
 
+    @classmethod
+    def get_holdings_by_document_by_view_code(cls, document_pid, viewcode):
+        """Returns holding pids by document and view code."""
+        all_holdings = []
+        es_query = HoldingsSearch()\
+            .filter('term', document__pid=document_pid)\
+            .source(['pid'])
+        if (viewcode != current_app.
+                config.get('RERO_ILS_SEARCH_GLOBAL_VIEW_CODE')):
+            org_pid = Organisation.get_record_by_viewcode(viewcode)['pid']
+            es_query.filter('term', organisation__pid=org_pid)
+        all_holdings = [result.pid for result in es_query.scan()]
```

Comment:
you can return the array directly, variable is useless

Reply:
done